jobserver/api.py

```diff
@@ -269,7 +270,25 @@ def get_queryset(self):
         # filter JobRequests by Backend name
         # Prioritise GET arg then self.backend (from authenticated requests)
         query_arg_backend = self.request.GET.get("backend", None)
+
+        # short term special case
+        if query_arg_backend == "nhsd":
+            query_arg_backend = "databricks"
+
         db_backend = getattr(self.backend, "name", None)
+
+        # send a warning to Sentry if the query arg and token-linked backend
+        # names differ
+        if query_arg_backend != db_backend:
```

Comment:
I think this needs an ` and db_backend is not None` clause on the end, otherwise accessing the API unauthenticated triggers a log message.